PlayCover/Model/PlayApp.swift

```diff
@@ -32,12 +33,72 @@ class PlayApp: BaseApp {
     }
     var sessionDisableKeychain: Bool = false
 
+    func fetchAppID(bundleID: String, completion: @escaping (String?) -> Void) {
```

Comment:
we already have a itunes lookup function, use that.
`PlayCover/Model/ITunesResponse.swift`

Reply:
ohhhh, tysm